File: app/src/main/java/ru/lazard/learnwords/BaseApplication.java
package ru.lazard.learnwords;

import android.app.Application;

import com.google.firebase.analytics.FirebaseAnalytics;

import ru.lazard.learnwords.model.Model;
import ru.lazard.learnwords.model.db.DbHelper;

/**
 * Created by Egor on 07.06.2016.
 */
public class BaseApplication extends Application{
    private static BaseApplication instance;

    public static BaseApplication getInstance() {
        return instance;
    }

    @Override
    public void onCreate() {
        instance = this;
        DbHelper.init(this);
        Model.init(this);

        super.onCreate();
        FirebaseAnalytics.getInstance(this);
    }
}

File: app/src/main/java/ru/lazard/learnwords/BaseApplication.kt
package ru.lazard.learnwords

import android.app.Application

import com.google.firebase.analytics.FirebaseAnalytics

import ru.lazard.learnwords.model.Model
import ru.lazard.learnwords.model.db.DbHelper
import ru.lazard.learnwords.ui.fragments.books.readBook.Translate

/**
 * Created by Egor on 07.06.2016.
 */
class BaseApplication : Application() {

    override fun onCreate() {
        instance = this
        DbHelper.init(this)
        Model.init(this)


        super.onCreate()
        FirebaseAnalytics.getInstance(this)
    }

    companion object {
        var instance: BaseApplication? = null
            private set
    }
}
